### PR TITLE
Added js / css animation question to general

### PR DESCRIPTION
**Pull request by isdampe**
There's a fair battle between CSS animation and javascript animation in the industry. Both have pro's and con's - would be good to know that a candidate is aware of more than one technique.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ The majority of the questions were plucked from an [oksoclap](http://oksoclap.co
 * What is FOUC? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
 * Explain what ARIA and screenreaders are, and how to make a website accessible.
+* Explain some of the pros / cons for both CSS animations VS javascript animating
```
Comment (darcyclarke) [inline README.md]: Could you change to (?): 
> Explain some of the pros and cons for CSS animations versus JavaScript animations

Just maybe small tweaks... but like the question for sure

Comment (appleboy) [inline README.md]: :+1: @darcyclarke

Comment (isdampe) [inline README.md]: Updated with changes requested :+1:

**Comment by appleboy** [comment]
@isdampe Thanks.

**Pull request merged**